portfolio/src/main/webapp/style.css

```diff
@@ -227,3 +226,34 @@ footer {
 #comment-container{
     margin: 50px;
 }
+
+.label {
+  margin-bottom: 10px;
+}
+
+#userComment {
+  background-color: rgb(240, 240, 240);
+  padding: 0px 15px;
+}
+
+commentorName {
+  font-size: 12pt;
+  padding-top: 5px;
+}
+
+#commentText {
+  padding-bottom: 10px;
+}
+
+.commentSection {
+  margin-bottom: 75px;
+  width: 100%;
+}
```

Comment:
nit: [separate words in ID and class names by a hyphen](https://google.github.io/styleguide/htmlcssguide.html#ID_and_Class_Name_Delimiters), they are usually in a lower case too. [Further reading](https://www.freecodecamp.org/news/css-naming-conventions-that-will-save-you-hours-of-debugging-35cea737d849/)

Reply:
Okay, thanks Mostafa!